lib/ransack.rb

```diff
@@ -8,6 +8,12 @@
 module Ransack
   extend Configuration
   class UntraversableAssociationError < StandardError; end;
+
+  SUPPORTS_ATTRIBUTE_ALIAS = begin
+    ActiveRecord::Base.respond_to?(:attribute_aliases)
+  rescue NameError
+    false
+  end
 end
 
```

Comment:
Do you know if there are versions of Active Record that do not support `attribute_aliases`?